.env

```diff
@@ -0,0 +1,6 @@
+VALID_EMAIL=lilkon50807@stud.noroff.no
```

Comment:
You should never commit an env file as this is a security risk. 

I notice that you don't have dotenv installed, and don't seem to be using this info in your testing?